docs/guide/app.md

```diff
@@ -182,6 +185,41 @@ Here's the question app with classvar CSS:
 --8<-- "docs/examples/app/question03.py"
 ```
 
+
+## Title and subtitle
+
+Textual applications have a `title` attribute that represents the name of your application and a `sub_title` attribute that gives additional context for it.
+These attributes can be displayed to the user if your application has a widget `Header` and there are two ways in which you can customize them if you don't want to run with the default values.
+
+By default, the title of your application matches the name of the application class and the subtitle is empty.
+However, you can override those defaults by setting the class attributes `TITLE` and `SUB_TITLE`, as shown below.
+Remember that you need the built-in widget `Header` to make the title and the subtitle visible in our app.
+
+```py title="question_title01.py" hl_lines="6-7 10"
+--8<-- "docs/examples/app/question_title01.py"
+```
+
+The app title and subtitle are displayed at the top of the application in the header:
+
+```{.textual path="docs/examples/app/question_title01.py"}
+```
+
+On top of being able to define a default title and a default subtitle for a given application class, each _instance_ of your application class has two attributes `title` and `sub_title` that can be used to modify the title and subtitle of said instance application.
+
+For example, the application shown below changes its title and subtitle as soon as the application is instantiated.
+
+```py title="question_title02.py" hl_lines="19-21"
+--8<-- "docs/examples/app/question_title02.py"
+```
+
+If you run the app shown above and if you press a key, the title and subtitle update accordingly.
+
+For example, if you press `t`, your application will look as shown below:
```

Comment:
We have an extension that renders keys, try replacing the key with `++t++`

Reply:
Thanks for the reminder o7